deploy/packages/bbp-packages.yaml

```diff
@@ -168,11 +168,12 @@ packages:
       - mpi
       - python
     specs:
-      - neurodamus-core~common@3.0.1
-      - neurodamus-neocortex+coreneuron@1.0
-      - neurodamus-hippocampus+coreneuron@1.0
-      - neurodamus-thalamus+coreneuron@1.0
-      - neurodamus-mousify+coreneuron@1.0
-      - neurodamus-neocortex+coreneuron@1.0^coreneuron+knl
-      - neurodamus-hippocampus+coreneuron@1.0^coreneuron+knl
-      - neurodamus-thalamus+coreneuron@1.0^coreneuron+knl
+      # Can we deploy always the latest? Versions are now bit more stable (and long)
```

Comment:
With the current implementation, this will generate modules for all neurodamus versions found.

Reply:
The next iteration of the deployment would solve this…

Reply:
last commit adds `@1.1-3.0.2` but the deployments scripts don't like it

Reply:
I'm not surprised...

Reply:
Actually it was due to the double `@@` typo :)
In the end, if you don't think is too bizarre I would like to go forward with it, and no worries, you can blame me for any problem, I will do my best at maintaining it. Just le me know if you see a better way at achieving a similar result, or you are happy with the implementation

Reply:
It still looks freakish to me, personally. Remove that comment above, and I'll approve, though. Technically nothing wrong with it…